Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -847,7 +875,7 @@ private void initUserSessionInternal(BranchReferralInitListener callback, Activi
         //If already initialised
         if (hasUser() && hasSession() && initState_ == SESSION_STATE.INITIALISED) {
             if (callback != null)
-                callback.onInitFinished(new JSONObject(), null);
+                callback.onInitFinished(getLatestReferringParams(), null);
```

Comment:
Hmm - this is a behavior change. The problem being that if developers call initSession when the state is already initialized, we didn't want to return the parameters over and over again. This is an issue when the developer uses initSession with callback in multiple Activities (which was common before auto session management). I can see how this would happen now with auto session management.

Can you check if auto session management, return getLatest, else, return new.